spec.emu

```diff
@@ -111,6 +111,36 @@ contributors: Ujjwal Sharma, Younies Mahmoud
       </table>
     </emu-table>
 
+    <emu-clause id="sec-todurationrecord" type="abstract operation">
+      <h1>
+        ToDurationRecord (
+          _input_: an ECMAScript language value
+        ): a Record with the fields listed in the Value Slot column of <emu-xref href="#table-duration-components"></emu-xref>
```

Comment:
```suggestion
        ): either a normal completion containing a Record with the fields listed in the Value Slot column of <emu-xref href="#table-duration-components"></emu-xref>, or an abrupt completion
```
(Sorry, I missed this the first time around)

Reply:
I had no idea you had to be explicit about this... is this real?

Reply:
Yes, it's for real! ecmarkup actually understands it and does some rudimentary type-checking.